src/core/Factory.sol

```diff
@@ -30,47 +30,52 @@ contract Factory is Controllable, ReentrancyGuardUpgradeable, IFactory {
 
     //region ----- Constants -----
 
-    /// @dev Version of Factory implementation
+    /// @inheritdoc IControllable
     string public constant VERSION = '1.0.0';
 
     uint internal constant _WEEK = 60 * 60 * 24 * 7;
 
     uint internal constant _PERMIT_PER_WEEK = 2;
 
+    // keccak256(abi.encode(uint256(keccak256("erc7201:stability.Factory")) - 1)) & ~bytes32(uint256(0xff));
+    bytes32 private constant FACTORY_STORAGE_LOCATION = 0x94b53192a2415b53b438d03f0efa946204c0118192627e3d5ed4ba034c9a0300;
+
     //endregion -- Constants -----
 
     //region ----- Storage -----
 
-    /// @inheritdoc IFactory
-    mapping(bytes32 typeHash => VaultConfig) public vaultConfig;
-    
-    /// @inheritdoc IFactory
-    mapping(bytes32 idHash => StrategyLogicConfig) public strategyLogicConfig;
-
-    /// @inheritdoc IFactory
-    mapping(bytes32 deploymentKey => address vaultProxy) public deploymentKey;
-
-    /// @inheritdoc IFactory
-    mapping(address vault => uint status) public vaultStatus;
-
-    mapping(address address_ => bool isStrategy_) public isStrategy;
-
-    /// @dev 2 slots struct
-    EnumerableSet.Bytes32Set internal _vaultTypeHashes;
-
-    /// @dev 2 slots struct
-    EnumerableSet.Bytes32Set internal _strategyLogicIdHashes;
+    /// @custom:storage-location erc7201:stability.Factory
+    struct FactoryStorage {
+        /// @inheritdoc IFactory
+        mapping(bytes32 typeHash => VaultConfig) vaultConfig;
+        /// @inheritdoc IFactory
+        mapping(bytes32 idHash => StrategyLogicConfig) strategyLogicConfig;
+        /// @inheritdoc IFactory
+        mapping(bytes32 deploymentKey => address vaultProxy) deploymentKey;
+        /// @inheritdoc IFactory
+        mapping(address vault => uint status) vaultStatus;
+        mapping(address address_ => bool isStrategy_) isStrategy;
+        /// @dev 2 slots struct
+        EnumerableSet.Bytes32Set vaultTypeHashes;
+        /// @dev 2 slots struct
```

Comment:
remove `/// @dev N slots struct` comments because it need only for gap size calculation. without gap its unnecessary